src/applications/disability-benefits/all-claims/components/ArrayField.jsx

```diff
@@ -0,0 +1,428 @@
+            const itemIdPrefix = `${idSchema.$id}_${index}`;
+            const itemIdSchema = toIdSchema(
+              itemSchema,
+              itemIdPrefix,
+              definitions,
+            );
+            const isLast = items.length === index + 1;
+            const isEditing = this.state.editing[index];
+
+            if (isEditing) {
+              return (
+                <div key={index} className="va-growable-background">
+                  <Element name={`table_${itemIdPrefix}`} />
+                  <div className="row small-collapse">
+                    <div className="small-12 columns va-growable-expanded">
+                      {isLast &&
+                      items.length > 1 &&
+                      uiSchema['ui:options'].itemName ? (
+                        <h5>New {uiSchema['ui:options'].itemName}</h5>
+                      ) : null}
+                      <div className="input-section">
+                        <SchemaField
+                          key={index}
+                          schema={itemSchema}
+                          uiSchema={uiSchema.items}
+                          errorSchema={
+                            errorSchema ? errorSchema[index] : undefined
+                          }
+                          idSchema={itemIdSchema}
+                          formData={item}
+                          onChange={value => this.onItemChange(index, value)}
+                          onBlur={onBlur}
+                          registry={this.props.registry}
+                          required={false}
+                          disabled={disabled}
+                          readonly={readonly}
+                        />
+                      </div>
+                      <div className="row small-collapse">
+                        <div className="small-6 left columns">
+                          {!isLast && (
+                            <button
+                              className="float-left"
+                              onClick={() => this.handleUpdate(index)}
+                            >
+                              Update
+                            </button>
+                          )}
+                          {isLast && (
+                            <button
+                              className="float-left"
+                              disabled={!this.props.formData}
+                              onClick={this.handleSave}
+                            >
+                              Save
+                            </button>
+                          )}
+                        </div>
+                        <div className="small-6 right columns">
+                          <button
+                            className="usa-button-secondary float-right"
+                            type="button"
+                            onClick={() => this.handleRemove(index)}
+                          >
+                            Remove
```

Comment:
What do you think of couching this in a `!isOnlyItem` or something so we don't show the remove button if it's the only item on the list (since the remove button won't do anything at that point)?